cli/helpers.py

```diff
@@ -124,22 +124,43 @@ def get_service_url_from_kubectl(service_name, cloud):
 
   raise Exception("Can't get ip address from cloud '{}'".format(cloud))
 
+def get_traffic_terraform_path(cloud):
+  path = "/terraform"
+  if 'aws' == cloud.lower():
+    path += "/aws/traffic"
+  elif 'azure' == cloud.lower():
+    path += "/azure/traffic"
+  elif 'gcp' == cloud.lower():
+    path += "/gcp/traffic"
+  else:
+    raise Exception("Cloud provider '{}' is NOT valid!".format(cloud))
+  return path
```

Comment:
nit: could be simplified to:

```python
def get_traffic_terraform_path(cloud):
    if not cloud:
      raise Exception("Cloud provider '{}' is NOT valid!".format(cloud))

    return "/terraform/{}/traffic".format(cloud.lower())
```

Reply:
This is close to what I was going for... but I do want to validate that something like cloud = 'foo' doesn't sneak by.

what about this instead:

```
def get_traffic_terraform_path(cloud):
   var valid_clouds = ['aws','azure','gcp'];
   if not valid_clouds.indexOf(cloud):
      raise Exception("Cloud provider '{}' is NOT valid!".format(cloud))

    return "/terraform/{}/traffic".format(cloud.lower())

```

Reply:
Ahhh got it; probably not worth the change then.  

the shorthand of that would be just:

```python
if not cloud or cloud.lower() not in ['aws','azure','gcp']:
```